server2/Arcadia.Assistant/Arcadia.Assistant.SF/ApplicationParameters/Cloud.xml

```diff
@@ -1,6 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Application xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="fabric:/Arcadia.Assistant.SF" xmlns="http://schemas.microsoft.com/2011/01/fabric">
   <Parameters>
+    <Parameter Name="Arcadia.Assistant.AppCenterBuilds_InstanceCount" Value="-1" />
+    <Parameter Name="Arcadia.Assistant.AppCenterBuilds_InstanceCount" Value="-1" />
     <Parameter Name="Arcadia.Assistant.PendingActions_InstanceCount" Value="-1" />
     <Parameter Name="Arcadia.Assistant.Vacations_InstanceCount" Value="-1" />
     <Parameter Name="Arcadia.Assistant.SickLeaves_InstanceCount" Value="-1" />
```

Comment:
lacks some parameters that are present in Local configuration

Reply:
Done